content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: blog
+title: "Kubernetes 1.24: Stargazer"
+date: 2022-05-03
+slug: kubernetes-1-24-release-announcement
+---
+
+# Kubernetes 1.24
+
+**Authors**: [Kubernetes 1.24 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/release-team.md)
+
+We are excited to announce the release of Kubernetes 1.24, the first release of 2022!
+
+This release consists of 46 enhancements: fifteen enhancements have graduated to stable, 
+fifteen enhancements are moving to beta, and thirteen enhancements are entering alpha. 
+Also, two features have been deprecated, and the [dockershim has been removed](/dockershim).     
+
+## Major Themes
+
+### Dockershim Removed from kubelet
+
+After its deprecation in v1.20, the dockershim component has been removed from the kubelet. 
+From v1.24 onwards, you will need to either use one of the other [supported runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes/) (such as containerd or CRI-O) 
```

Comment:
```suggestion
From v1.24 onwards, you will need to either use one of the other [supported runtimes](/docs/setup/production-environment/container-runtimes/) (such as containerd or CRI-O) 
```

Reply:
As these are CNCF projects, should we link to their websites?
```suggestion
From v1.24 onwards, you will need to either use one of the other [supported runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes/) (such as [containerd](https://containerd.io/) or [CRI-O](https://cri-o.io/)) 
```